src/lib/app/qzcommon.cpp

```diff
@@ -30,4 +30,5 @@ QUPZILLA_EXPORT const char* AUTHOR = "David Rosca";
 QUPZILLA_EXPORT const char* COPYRIGHT = "2010-2014";
 QUPZILLA_EXPORT const char* WWWADDRESS = "http://www.qupzilla.com";
 QUPZILLA_EXPORT const char* WIKIADDRESS = "https://github.com/QupZilla/qupzilla/wiki";
+QUPZILLA_EXPORT const char* RAWVCS = "https://raw.githubusercontent.com/QupZilla/qupzilla/v1.8";
```

Comment:
Remove the url here and move it to cabundleupdater.cpp - preferably as 
```cpp
static const char* CA_BUNDLE_URL = "https://raw.githubusercontent.com/QupZilla/qupzilla/v1.8/src/lib/data/data/bundle_version";
```
and then it's good to go.